spynnaker/pyNN/models/neuron/synapse_dynamics/synapse_dynamics_structural_static.py

```diff
@@ -17,13 +17,15 @@
 from .abstract_synapse_dynamics_structural import (
     AbstractSynapseDynamicsStructural)
 from .synapse_dynamics_structural_common import (
-    SynapseDynamicsStructuralCommon as
-    CommonSP)
+    SynapseDynamicsStructuralCommon as CommonSP)
 from .synapse_dynamics_static import SynapseDynamicsStatic
+from .synapse_dynamics_stdp import SynapseDynamicsSTDP
+from .synapse_dynamics_structural_stdp import SynapseDynamicsStructuralSTDP
+from spynnaker.pyNN.exceptions import SynapticConfigurationException
 
 
-class SynapseDynamicsStructuralStatic(AbstractSynapseDynamicsStructural,
-                                      SynapseDynamicsStatic):
+class SynapseDynamicsStructuralStatic(
+        SynapseDynamicsStatic, AbstractSynapseDynamicsStructural):
     """ Class that enables synaptic rewiring. It acts as a wrapper\
```

Comment:
Class that enables synaptic rewiring in the absence of STDP.

Reply:
Fixed.